CHANGELOG.md

```diff
@@ -2,18 +2,20 @@
 
 All notable changes to this project will be documented in this file.
 
-## 2.1.0 (Unreleased)
+## 2.2.0 (unreleased)
+
+### Breaking changes
+
+- `chainName` field was replaced by `chain` in multiple interfaces
```

Comment:
Minor: Probably a good idea to mention the interfaces affected instead of saying `multiple`